<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMerger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="geometryTypeLabel">
        <property name="text">
         <string>Geometry type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbGeometryType"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="InputGroupBox">
     <property name="title">
      <string>Input layers</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="cmbSourceType"/>
      </item>
      <item>
       <widget class="QWidget" name="pnlDir" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lbDirName"/>
         </item>
         <item>
          <widget class="QToolButton" name="btnSelectDir">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pnlFiles" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Select files:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnSelectFiles">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tblInputLayers"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="OutputGroupBox">
     <property name="title">
      <string>Output layer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QToolButton" name="btnOutSelectLayer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="leOutLayer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>memory:</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Encoding:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Spatial reference:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QgsProjectionSelectionWidget" name="cmbOutSpatialReference"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cmbOutEncoding"/>
      </item>
     </layout>
     <zorder>leOutLayer</zorder>
     <zorder>cmbOutEncoding</zorder>
     <zorder>btnOutSelectLayer</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>cmbOutSpatialReference</zorder>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="AdvancedGroupBox">
     <property name="title">
      <string>Advanced settings</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="chkAddFileName">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add field with file name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAddFilePath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add field with file path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAddResultToMap">
        <property name="text">
         <string>Add result layer to map canvas</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QComboBox</extends>
   <header location="global">qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
